tests/route/test_static_route.py

```diff
@@ -248,6 +250,11 @@ def run_static_route_test(duthost, unselected_duthost, ptfadapter, ptfhost, tbin
                 config_reload(duthost, wait=500)
             else:
                 config_reload(duthost, wait=450)
+            # In case of dualtor, config_reload can result in a switchover (active tor can become standby).
+            # So we need to make sure rand_selected_dut is in active state before verifying traffic.
+            if is_dual_tor:
```

Comment:
It is not recommended to import/use functions starting with underscore.
We could simply do a `config mux mode active all; config mux mode auto all` here to do the toggle via cmdline.

Reply:
Right, wanted to check for the better way to handle this. `config mux mode active all; config mux mode auto all` this doesn't seem to be helping here (to toggle ports to active). Will try to check bit more on this and update. Thanks!

Reply:
Done, updated the diff. (Verified that test is passing)

Reply:
We still need to config the mux back to `auto`, I didn't see the logic here

Reply:
That will be done as part of fixture `toggle_all_simulator_ports_to_rand_selected_tor_m ` teardown (https://github.com/sonic-net/sonic-mgmt/blob/202311/tests/common/dualtor/mux_simulator_control.py#L670)

Reply:
looks this fixture would exit early due to no `active-standby` mux ports present on `dualtor-aa` topology.
https://github.com/sonic-net/sonic-mgmt/blob/ea61f075b77f1d79a678033de06edb7cba11659c/tests/common/dualtor/mux_simulator_control.py#L656-L660

Better to config mux back to `auto` here.

Reply:
This fixture will be executed only for `active-standby` topologies (as it's exiting early for `active-active` topo).
And we have a seperate fixture `setup_standby_ports_on_rand_unselected_tor` executed for `active-active` topo's, this also restores the mux config back to `auto` https://github.com/sonic-net/sonic-mgmt/blob/202311/tests/common/dualtor/dual_tor_utils.py#L1644.

Reply:
Slightly irrevelant to this PR changes, but I think it's better to do `config save -y` during teardown of `setup_standby_ports_on_rand_unselected_tor` (as we are do for `toggle_all_simulator_ports_to_rand_selected_tor_m`), because for tests like this (performing config save and reload) config_db.json would be updated with fixture setup changes but teardown changes won't get into config_db.json. I can float seperate review for this change as it's not relevant to this PR changes.